manifests/z/zumoshi/BrowserSelect/1.4.1/zumoshi.BrowserSelect.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: zumoshi.BrowserSelect
+PackageVersion: 1.4.1
+PackageLocale: en-US
+Publisher: zumoshi
+PublisherUrl: https://github.com/zumoshi
+Author: zumoshi
+PackageName: BrowserSelect

```

Comment:
```suggestion
PackageName: BrowserSelect -- select browser dynamically
```

Reply:
This key must match Control Panel according to the latest documentation for creating manifests.